crypto/bio/bss_conn.c

```diff
@@ -438,12 +446,13 @@ static long conn_ctrl(BIO *b, int cmd, long num, void *ptr)
     case BIO_C_SET_CONNECT:
         if (ptr != NULL) {
             b->init = 1;
-            if (num == 0) {
+            if (num == 0) { /* BIO_set_conn_hostname */
```

Comment:
Thanks for the comments.

This part would be good to have in the 1.1.1 branch.

Reply:
Argh, I just did this backport and by mistake directly pushed it to `OpenSSL_1_1_1-stable` :(
See #12034.

Reply:
The actual backport PR is now in #12036.